README.md

```diff
@@ -8,20 +6,23 @@
 [![codecov](https://codecov.io/gh/JuliaDynamics/Agents.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/JuliaDynamics/Agents.jl)
 [![Package Downloads](https://shields.io/endpoint?url=https://pkgs.genieframework.com/api/v1/badge/Agents)](https://pkgs.genieframework.com?packages=Agents)
 
-Agents.jl is a [Julia](https://julialang.org/) framework for agent-based modeling (ABM). All further information about Agents.jl are provided in the docs.
+Agents.jl is a pure [Julia](https://julialang.org/) framework for agent-based modeling (ABM): a computational simulation of autonomous agents that react to their environment (including other agents) given a predefined set of rules.
+Some major highlights of Agents.jl are:
 
-## Contributions
+1. It is fast (faster than MASON, NetLogo, or Mesa)
+2. It is simple: has a very short learning curve and requires writing minimal code
+3. Has an extensive interface of thousands of out-of-the box possible agent actions
+4. Straightforwardly allows simulations on Open Street Maps
 
-Any contribution to Agents.jl is welcome! For example you can:
+The simplicity of Agents.jl is due to the intuitive space-agnostic modelling approach we have implemented: agent actions are specified using generically named functions (such as "move agent" or "find nearby agents") that do not depend on the actual space the agents exist in, nor on the properties of the agents themselves. Overall this leads to ultra fast model prototyping where even changing the space the agents live in is matter of only a couple of lines of code.
```

Comment:
Don't want to sound too picky but the reference only to its simplicity seems kind of strange to me :D I like more the one here https://juliadynamics.github.io/Agents.jl/previews/PR817/:

"Besides its incredible performance, two more major highlights of Agents.jl are its simplicity and the total amount of features. Agents.jl is designed to require writing minimal code and being very simple to learn and use, even though it provides an extensive API with thousands of possibilities for agent actions. This simplicity is due to the intuitive space-agnostic modelling approach we have implemented: agent actions are specified using generically named functions that do not depend on the actual space the agents exist in or on the properties of the agents themselves. Overall this leads to ultra fast model prototyping where even changing the space of the agents is matter of only a couple of lines of code."

Reply:
But maybe it's a bit repetitive this way given that the bullet points already address those aspects at the same level of details...

Reply:
it is repetitive.